packages/desktop/views/dashboard/collectibles/views/DetailsView.svelte

```diff
@@ -52,6 +55,10 @@
             storageDeposit: { data: String(storageDeposit) },
         }),
     }
+
+    function handleExplorerClick(): void {
+        Platform.openUrl(`${explorerUrl}/nft/${id}`)
```

Comment:
Can we add the nft endpoint to a constant? Perhaps we need an enum for the different endpoints.